src/appMain/CMakeLists.txt

```diff
@@ -47,16 +48,21 @@ include_directories(
   ${COMPONENTS_DIR}/smart_objects/include
   ${COMPONENTS_DIR}/media_manager/include
   ${COMPONENTS_DIR}/telemetry_monitor/include
-  ${POLICY_PATH}/include/
+  ${COMPONENTS_DIR}/functional_module/include
+  ${POLICY_PATH}/include
   ${POLICY_GLOBAL_INCLUDE_PATH}/
-  ${COMPONENTS_DIR}/rpc_base/include/
+  ${COMPONENTS_DIR}/rpc_base/include
   ${COMPONENTS_DIR}/resumption/include
   ${COMPONENTS_DIR}/dbus/include
   ${CMAKE_BINARY_DIR}/src/components
   ${JSONCPP_INCLUDE_DIRECTORY}
   ${LOG4CXX_INCLUDE_DIRECTORY}
   ${OPENSSL_INCLUDE_DIRECTORY}
   ${MESSAGE_BROKER_INCLUDE_DIRECTORY}
+  ${ENCRYPTION_INCLUDE_DIRECTORY}
```

Comment:
@dev-gh removed in https://github.com/LuxoftAKutsan/sdl_core/pull/113/commits/69b0106c66b759cb7e1c5b56cd36e7c35b1a4867